src/lib/hooks/useMint.ts

```diff
@@ -121,7 +121,7 @@ export const useMint = (): UseMintReturn => {
 			//////////////////////////
 
 			setStatus(
-				'Minting your Pet... this may take up to 20 minutes if the network is busy. You may close this and the transaction will continue in the background. When minting is complete, your Pet will be in your profile.',
+				'Minting your Beast... this may take up to 20 minutes if the network is busy. You may close this and the transaction will continue in the background. When minting is complete, your Beast will be in your profile.',
```

Comment:
Maybe change `your Beast will be in your profile` to `your Beast(s) will be in your profile`